app/assets/stylesheets/application.css.scss

```diff
@@ -18,3 +18,163 @@
 @import "bootstrap";
 @import "font-awesome-sprockets";
 @import "font-awesome";
+
+body {
+  font-family: "century gothic", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-size: 14px;
+  line-height: 1.42857143;
+  color: #333333;
+  background-color: #ebe7df
+}
+
+.row h1 {
+  font-weight:bold;
+  font-size:20px;
+  background:#418a95;
+  padding:30px 10px;
+  color:#fff;
+  margin-bottom:0;
+}
+
+.table {
+  width: 100%;
+  max-width: 100%;
+  margin-bottom: 20px;
+  box-shadow:0px 10px 10px rgba(0, 0, 0, 0.2);
+}
+.table tr {
+  border-top:1px solid #ebe7df;
+}
+.table thead tr {
+  border-bottom:1px solid #ebe7df;
+  border-top:1px solid #63acb7;
+  background:#63acb7;
+  color:#fff;
+}
+.table thead tr:first-child {
+  border-top:none
+}
+.table tbody {
+  background:#ccc;
+}
+tr:nth-child(even) td { background:linear-gradient(to bottom, #f2f1e8 0%,#f6f7ea 100%);}
+
+/* Cells in odd rows (1,3,5...) are another (excludes header cells)  */
+tr:nth-child(odd) td { background: #FEFEFE; }
+
+.table>thead>tr>th,
+.table>thead>tr>td,
+.table>tbody>tr>th,
+.table>tbody>tr>td,
+.table>tfoot>tr>th,
+.table>tfoot>tr>td {
+  padding: 10px;
+  line-height: 1.42857143;
+  vertical-align: top;
+}
+
+.table>thead>tr>th {
+  vertical-align: bottom;
+  border-bottom: 2px solid #ddd;
+  text-transform:uppercase;
+}
+
+.table>thead>tr>th {
+  vertical-align: bottom;
+  border-bottom: 2px solid #ddd;
+  text-transform:uppercase;
+}
+
+.btn-primary {
+  color: #fff;
+  background-color: #535353;
+  border-color: #ccc;
+  border-radius:0px;
```

Comment:
Colon after property should be followed by one space<br>`0px` should be written without units as `0`